addons/website_slides/controllers/main.py

```diff
@@ -1321,7 +1321,7 @@ def slide_category_search_read(self, fields, domain):
         category_slide_domain = expression.AND([category_slide_domain, [('is_category', '=', True)]])
         can_create = request.env['slide.slide'].check_access_rights('create', raise_exception=False)
         return {
-            'read_results': request.env['slide.slide'].search_read(category_slide_domain, fields),
+            'read_results': request.env['slide.slide'].search_read(category_slide_domain, fields, order='id'),
```

Comment:
can we order by `create_date, id` ? (double check if it need ASC or DESC)

Reply:
I think it's a bit better yes, it cost almost nothing (and it's technically possible to resequence, etc)